### PR TITLE
Sprint 3 requirements

### PR DESCRIPTION
**Pull request by pjanck**
## Miscellaneous

Initiating requirements for sprint 3 topics.

### CODE REVIEW

**Review by SergejMuhic — COMMENTED** [review]
Just clicking through for now.

File: sprints/sprint3_geo.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+1. `IfcGeoScienceFeature`
+    1. [part of](#new-concepts) `IfcGeoScienceModel/GEOTECHMODEL`
+    1. `PredefinedType` - at least one of:
+
+    1. `PHYSICALPROPERTYDISTRIBUTION`
+        1. [body geometry](./sprint2_3.md) 
+            1. If geometry = IfcVoxelGrid an associated instance of IfcVoxelData (new CT!) is required
+        1. [local placement](./sprint2_2.md)
+        1. [properties](./sprint2_3.md) with `Pset_GeoScienceFeatureTypePropertyDistribution`
```
Comment (larswik) [inline sprints/sprint3_geo.md:119]:
```suggestion
        1. [properties](./sprint2_3.md) with `Pset_GeoScienceFeatureCommon`
```

**Review by SergejMuhic — APPROVED** [review]
Unter Vorbehalt!

**Comment by pjanck** [comment]
> You is in conflict!

Not anymore I am.

**Pull request merged**